spec/dummy/app/themes/default/views/layouts/thredded.html.erb

```diff
@@ -9,7 +9,16 @@
 
   <%= content_tag(:body, id: (yield :thredded_page_id), class: 'layout-application') do %>
     <%= render 'thredded/shared/icons_svg' %>
-    <%= yield %>
+
+    <div class="main-container">
+      <header class="navigation" role="banner">
+        <%#= render 'thredded/shared/top_nav' %>
+        <%#= render 'thredded/shared/topic_nav' %>
```

Comment:
I'm not sure I understand. These partials are now (inline) in the layout?

If that's the case then feel free to delete the partials completely if they're not being used.